fasta/multiFa.go

```diff
@@ -7,19 +7,25 @@ import (
 	"strings"
 )
 
+//RefPosToAlnPos returns the alignment position associated with a given reference position for an input MultiFa. 0 based.
 func RefPosToAlnPos(record Fasta, RefPos int) int {
-	var AlnPos int = 0
-	var refCounter int = 0
+	return RefPosToAlnPosCounter(record, RefPos, 0, 0)
+}
 
-	for t := 0; refCounter < RefPos; t++ {
-		AlnPos++
+//RefPosToAlnPosCounter is like RefPosToAlnPos, but can begin midway through a chromosome at a refPosition/alnPosition pair, defined by the input variables refStart and alnStart.
+func RefPosToAlnPosCounter(record Fasta, RefPos int, refStart int, alnStart int) int {
+	if refStart > RefPos {
+		refStart, alnStart = 0, 0 //in case the refStart was improperly set (greater than the desired position, we reset these counters to 0.
```

Comment:
Could you explain why we wouldnt want/it doesn't work for refStart to be greater than RefPos? And why we would want to reset the alnStart if the refStart is greater than RefPos? I'm thinking mid-way through a chromosome these start positions would always be greater (if we were on the positive sense).